examples/java/spring-boot/manifest.yml

```diff
@@ -0,0 +1,7 @@
+---
+applications:
+  - name: federated-api-model-spring-boot
```

Comment:
can we get this renamed (+ in PAAS) to `federated-api-model-spring-boot-sandbox`?